### PR TITLE
Refresh plugin and depends on mysql-api-plugin

### PR DESCRIPTION
**Pull request by jonesbusy**
Refresh plugin for November 2023

Need admin to restart build

Depends on https://github.com/jenkinsci/mysql-api-plugin/pull/4/files

### Testing done

```
mvn clean install
```

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,16 +28,27 @@
...
+    </dependencyManagement>
+
   <dependencies>
     <dependency>
-      <groupId>mysql</groupId>
-      <artifactId>mysql-connector-java</artifactId>
-      <version>8.0.33</version>
+        <groupId>io.jenkins.plugins</groupId>
+        <artifactId>mysql-api</artifactId>
+        <version>8.0.16</version>
```
Comment (jonesbusy) [inline pom.xml:47]: https://github.com/jenkinsci/mysql-api-plugin/pull/4/files

**Comment by jonesbusy** [comment]
Hi,

@timja Do you wish I add CD also to the plugin ?

Regards,

**Comment by timja** [comment]
> Hi,
> 
> @timja Do you wish I add CD also to the plugin ?
> 
> Regards,

Yes please, do you want commit access?

**Comment by jonesbusy** [comment]
> > Hi,
> > @timja Do you wish I add CD also to the plugin ?
> > Regards,
> 
> Yes please, do you want commit access?

Sure done here https://github.com/jenkins-infra/repository-permissions-updater/pull/3636

**Pull request merged**